### PR TITLE
perf(log): :zap: improve the log output

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
1. add more messages to make the output more clear.
2. use `__non_webpack_require__` to instead of `eval('require')`.

Ref #102

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import CommitProvider from './lib/editor/provider';
 
 export function activate(context: vscode.ExtensionContext) {
   output.initialize();
-  output.appendLine('Activated');
+  output.appendLine('[info] Extension Activated');
```
Comment (vivaxy) [inline src/extension.ts:13]: We can refactor `output` to have `output.info`, `output.warning` and `output.error` APIs

Comment (yi-Xu-0100) [reply src/extension.ts:13]: @vivaxy I will try to refactor it.

**Comment by yi-Xu-0100** [comment]
@vivaxy I refactor the commit history. 😁

See in https://github.com/vivaxy/vscode-conventional-commits/pull/104/commits

**Comment by vivaxy** [comment]
@yi-Xu-0100 Thank you! It's a nice enhancement.

**Review by vivaxy — APPROVED** [review]

**Pull request merged**